coordinape/coordinape
# Create Org Settings Page

Create a new Settings page for Organizations:
- supports changing org name, telegram_id, and logo.
- add link to settings page for Orgs page
<img width="1342" alt="Screen Shot 2022-10-07 at 5 40 14 PM" src="https://user-images.githubusercontent.com/83605543/194678858-b2e68016-606c-4733-8cd2-ff2dc1f5de52.png">

<img width="675" alt="Screen Shot 2022-10-07 at 5 39 35 PM" src="https://user-images.githubusercontent.com/83605543/194678863-cd0aead0-3d6a-4478-9e5c-52bd7cc89035.png">



## Testing:

- File uplaod on all pages including profiles, as changed some of that logic
- Test form behavior
- Check migrations

TODOs:
- [ ] verify the contraints will be valid against existing prod data


Changes:
- Remove unneccessary Box in SavingIndicator
- Add migrations with db constraints
- update org permissions
- create org settings page
- add routes
- fix express file upload size limit
- update types
- Remove unneeded useEffect
- Add error handling to uploadOrgLogo
